### PR TITLE
Fix hugo deprecation warning

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a deprecation warning emitted by hugo:

```
WARN  deprecated: config: languages.en.description: custom params on the language top level
was deprecated in Hugo v0.112.0 and will be removed in a future release. Put the value below
[languages.en.params]. See https://gohugo.io/content-management/multilingual/#changes-in-hugo-01120
```

### CODE REVIEW

**Comment by vinooganesh** [comment]
Hi @deining - this is actually fixed on staging and there will be a pretty large bulk change going to production soon: https://github.com/apache/parquet-site/blob/staging/config.toml#L50. I'd recommend waiting until that change has merged, because many issues have been fixed there.

**Review by wgtmac — APPROVED** [review]

**Pull request merged**
